client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/tailored-flow-precheckout-screen/index.tsx

```diff
@@ -81,10 +87,14 @@ export default function TailoredFlowPreCheckoutScreen( { flowName }: { flowName:
 		<div className="processing-step__container">
 			<div className="processing-step">
 				<h1 className="processing-step__progress-step">{ steps.current[ currentStep ]?.title }</h1>
-				<LoadingBar
-					className="processing-step__progress-bar"
-					progress={ ! hasStarted ? /* initial 10% progress */ 0.1 : progress }
-				/>
+				{ isNewsletterOrLinkInBioFlow( flowName ) ? (
+					<LoadingBar
+						className="processing-step__progress-bar"
+						progress={ ! hasStarted ? /* initial 10% progress */ 0.1 : progress }
+					/>
+				) : (
+					<LoadingEllipsis />
+				) }
```

Comment:
We have two versions of the processing screen. The following line in the main file will load this one (tailored-flow-precheckout-screen/index.tsx) only for certain flows. So here, within this file, we just need to ensure we load ellipsis for the relevant flows. 

https://github.com/Automattic/wp-calypso/blob/f02f1d992ade8d495b2f7ee252053ddf47b27f1e/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx#L126